docker/testenv/Dockerfile

```diff
@@ -0,0 +1,69 @@
+#==============================================================================
+# Docker image for a Huntsman testing environment
+#==============================================================================
+
+FROM gcr.io/panoptes-exp/pocs:latest
+
+ENV PANDIR_OLD=$PANDIR
+ENV PANUSER=huntsman
+ENV PANDIR=/var/$PANUSER
+ENV POCS=$PANDIR/POCS
+ENV HUNTSMAN_POCS=${PANDIR}/huntsman-pocs
+
+# Install required packages
+RUN apt-get update && \
+    apt-get -y install build-essential && \
+    apt-get -y install byobu sshfs vim grc sudo && \
+    apt install -y openssh-server --fix-missing && \
+    apt install -y openssh-client && \
+
+    # Setup PANUSER
+    useradd -m -d /home/$PANUSER $PANUSER && \
+    echo $PANUSER:$PANUSER | chpasswd && \
```

Comment:
What's going on here?  The `useradd` function should be able to set a default password.